docs/discord/discord-readme.md

```diff
@@ -93,18 +93,23 @@ Tips to help you ask questions that will give you a better chance to get the bes
 3. **Clearly describe the problem.** Explain what is happening and what you want to happen.
 4. **Share your code.** It’s very unlikely that anyone will be able to solve your problem without seeing your code. Share a GitHub repository, a GitHub gist link, or format the code by wrapping it in backticks, like the example below.
 
-\```c++
+````txt
+```ts
   // Your code goes here
-\``` <!-- In Discord, this will look fine -->
+```
+````
+
+<!-- markdownlint-disable ol-prefix -->
```

Comment:
The codeblock breaks the list in the linter's eyes, so need to temporarily ignore it.